assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -493,9 +493,12 @@ call_catalina () {
     fi
   fi
 
+  ### Initialize default JVM arguments to run che
+  [ -z "${JAVA_OPTS}" ]  && JAVA_OPTS="-Xms256m -Xmx2048m  -Djava.security.egd=file:/dev/./urandom"
+
   ### Cannot add this in setenv.sh.
   ### We do the port mapping here, and this gets inserted into server.xml when tomcat boots
-  export JAVA_OPTS="${JAVA_OPTS} -Dport.http=${CHE_PORT}"
+  export JAVA_OPTS="${JAVA_OPTS} -Dport.http=${CHE_PORT} -Dche.home=${CHE_HOME}"
```

Comment:
To use setenv.sh from che-lib.And because che.home is che only specific variable.